### PR TITLE
Flask server code update version1

### LINKED ISSUE #3: Auto Scaler logic enhancement
Need to tweak autoscaler logic to improve accuracy

### PR DESCRIPTION
**Pull request by ShriramShagri**

### CODE REVIEW

**Comment by ShriramShagri** [comment]
@shree-vidya Merge code with final code on `kube.py` file before merging this.

File: coxswain/kube.py [hunk 1]
```diff
@@ -87,31 +88,33 @@ def getDeploymentInfo(self, name : str, namespace : str) -> dict:
...
-                "containerName" :  i.spec.containers[0].name,
-                "containerImage" : i.spec.containers[0].image, 
-                "maxContainerCpuLimit":i.spec.containers[0].resources.limits.cpu , 
-                "maxContainerMemoryLimit" : i.spec.containers[0].resources.limits.memory,
-                "minContainerCpu" : i.spec.containers[0].resources.requests.cpu,
-                "minContainerMemory" : i.spec.containers[0].resources.requests.memory
-            }
-            ans.append(d)
+        for i in ret.items:
+            if(i.metadata.namespace != "kube-system"):
```
Comment (ShriramShagri) [inline coxswain/kube.py]: Why brackets u cpp developer @shree-vidya 😂

**Comment by ShriramShagri** [comment]
Exception yet to be handled

**Comment by ghost** [comment]
Hello

**Pull request merged**